prow/plugins/releasenote/releasenote.go

```diff
@@ -43,16 +43,13 @@ const (
 	releaseNoteNone           = "release-note-none"
 	releaseNoteActionRequired = "release-note-action-required"
 
-	releaseNoteFormat       = `Adding %s because the release note process has not been followed.`
-	releaseNoteSuffixFormat = `One of the following labels is required %q, %q, or %q.
-Please see: https://git.k8s.io/community/contributors/devel/pull-requests.md#write-release-notes-if-needed.`
+	releaseNoteFormat       = `Adding the "%s" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/contributors/guide/release-notes.md) to remove it.`
```

Comment:
broken link. one too many contributors/ in it

Reply:
🤦‍♂️  good catch